dotnet/src/Agents/OpenAI/Internal/AssistantThreadActions.cs

```diff
@@ -401,7 +401,7 @@ public static async IAsyncEnumerable<StreamingChatMessageContent> InvokeStreamin
                     switch (contentUpdate.UpdateKind)
                     {
                         case StreamingUpdateReason.MessageUpdated:
-                            yield return GenerateStreamingMessageContent(agent.GetName(), contentUpdate);
+                            yield return GenerateStreamingMessageContent(agent.GetName(), run!, contentUpdate, logger);
```

Comment:
Ideally we should send as `innerContent` the raw representation/sdk original update.
Suggest using the `named parameters`, to make it clear which is which.

```suggestion
                            yield return GenerateStreamingMessageContent(agent.GetName(), update, contentUpdate, logger);
```

Reply:
Ok, yes.  Good point.  A couple notes.  `contentUpdate` is just a down-cast reference to the same instance as `update` ... and neither have a `Run` property that I can use for the parameter you're suggesting to remove.  I think I can just add `contentUpdate` as `InnerContent`